Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/subdisc/HL7PRPA201306Transforms.java

```diff
@@ -203,9 +209,9 @@ public PRPAIN201306UV02 createPRPA201306ForErrors(PRPAIN201305UV02 oRequest, Str
         // If message fails to leave local community then senderid and receiverid will match
         try {
             LOG.info("Attempting to retrieve property: " + NhincConstants.HOME_COMMUNITY_ID_PROPERTY
-                + " from property file: " + NhincConstants.GATEWAY_PROPERTY_FILE);
+                    + " from property file: " + NhincConstants.GATEWAY_PROPERTY_FILE);
```

Comment:
That might be good.  Except we should ignore this change because line wrapping was done from the auto-formatting in eclipse.  I didn't intentionally make that change.